src/librustdoc/src/lib.rs

```diff
@@ -741,30 +740,35 @@ where R: 'static + Send,
             }
         }
 
-        // Load all plugins/passes into a PluginManager
-        let path = plugin_path.unwrap_or("/tmp/rustdoc/plugins".to_string());
-        let mut pm = plugins::PluginManager::new(PathBuf::from(path));
+        if !plugins.is_empty() {
+            eprintln!("WARNING: --plugins no longer functions; see CVE-2018-1000622");
+        }
+
+        if !plugin_path.is_none() {
+            eprintln!("WARNING: --plugin-path no longer functions; see CVE-2018-1000622");
+        }
+
+        info!("Executing passes");
+
         for pass in &passes {
-            let plugin = match passes::PASSES.iter()
+            // determine if we know about this pass
+            let pass = match passes::PASSES.iter().find(|(p, ..)| p == pass) {
+            /*
                                              .position(|&(p, ..)| {
                                                  p == *pass
                                              }) {
-                Some(i) => passes::PASSES[i].1,
+                                                 */
```

Comment:
We can probably scrap this zombie code, yeah?

Reply:
Ugh, yes, I meant to and forgot. :)